search_blackbox_test.go

```diff
@@ -28,6 +30,16 @@ import (
 	"golang.org/x/net/context"
 )
 
+var spaceBlackBoxTestConfiguration *config.ConfigurationData
```

Comment:
Suggestion: @sbose78 this is code is added again and again. We could avoid it by simply having in a common place e.g `github.com/almighty/almighty-core/test/configuration/<A_NAME>.go`. Then you import it in each test where is required using `underscore`

```
import _ "github.com/almighty/almighty-core/test/configuration" 
...
```
that will run the init before the init of the own test. That should work but give it a try. Once you have it you can also get the Configuration variable via an exported function. e.g. `configuration.ConfigurationData()`